SPECS/kernel/CVE-2022-1012.nopatch

```diff
@@ -0,0 +1 @@
+CVE-2022-1012 - patched in 5.10.119
```

Comment:
nit - technically we skipped 5.10.119 in Mariner and went straight to 5.10.123.1
```suggestion
CVE-2022-1012 - patched in 5.10.123.1
```

Reply:
Thank you, updating.